app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -100,6 +100,11 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
         getHeaderViewModel().profile.set(it)
         getFooterViewModel().isAdmin.set(it.isAdmin)
         binding.administratorControlsLinearLayout.setOnClickListener {
+          if (getFooterViewModel().isAdministratorControlsSelected.get() == true) {
```

Comment:
isn't `if (getFooterViewModel().isAdministratorControlsSelected.get()) {` this is good to go?

Reply:
No, in that case !! operator will be required.

```
if (getFooterViewModel().isAdministratorControlsSelected.get()!!) {

}
```